src/compiler/circuit_error.zk

```diff
@@ -0,0 +1,32 @@
+machine fibo(signal n) (signal b: field) {
```

Comment:
perhaps ".chiquto" and put it in another folder like ./test or name the file `test_`